ee-9/pom.xml

```diff
@@ -73,6 +73,19 @@
         <!--<version.jakarta.ws.rs.jakarta-ws-rs-api>3.0.0</version.jakarta.ws.rs.jakarta-ws-rs-api>-->
         <!--<version.jakarta.xml.bind.jakarta-xml-bind-api>3.0.0</version.jakarta.xml.bind.jakarta-xml-bind-api>-->
         <version.org.apache.myfaces.core>3.0.1</version.org.apache.myfaces.core>
+        <!--
+            Overrides for MP5. Also make sure to update these in testsuite/integration/microprofile-tck/pom.xml
```

Comment:
Please check if Brians remark to my PR also applies here, for all overrides: https://github.com/wildfly/wildfly/pull/14855#discussion_r743986542

Reply:
@boris-unckel I verified the module.xml resources for a lot (not all though!) of these contain the expected artifacts.

Reply:
Yeah, it is slightly mysterious:-) But I was very happy to see the least effort approach work ;-)